.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+### All submissions:
+
+* [ ] Does your code follow
+  the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
+* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
+
+<!-- Mark completed items with an [x] -->
+
+<!-- You can erase any parts of this template not applicable to your Pull Request. -->
+
+### Changes proposed in this Pull Request:
+
+<!-- Describe the changes made to this Pull Request and the reason for such changes. -->
+
+Closes # .
+
+### How to test the changes in this Pull Request:
+
+<!-- Add detailed instructions for how to test that this PR fixes the issue and confirm that it doesn't break any other features :) -->
+
+1.
+2.
+3.
+
+### Changelog entry
+
+<!-- Add suggested changelog entry here. For example: -->
+> Fix / Tweak / Dev - Remove duplicate visibility meta entries from postmeta table
```

Comment:
Great idea adding the changelog "categories" here in the prompt - should encourage consistency. 

These are my preferred changelog categories - but this is a team decision, this is just my personal opinion.

- `New` for new features
- `Fix` for fixes to existing features
- `Dev` for anything that doesn't fit into the other categories

Reply:
I added this based on the categories we're using in _changelog.txt_. I Would like to hear what others in @woocommerce/tetris think?

Reply:
woocommerce changelog.txt uses: Add, Dev, Enhancement, Fix, Tweak, Update, Features - quite a lot actually.
This is not very important. Using the ones identified by @rcstr makes a lot of sense since they are already in the changelog.

Reply:
I'll just add _why_ I dislike tweak –  but happy for it to stick around if that's where we end up. I don't like `Tweak` because it doesn't indicate why the change was made, or who should pay attention. It's almost like a category for things that should not be in changelog! Personally I care a lot about changelogs indicating why stuff happened - what the intention was at the time. `Tweak` undermines that IMO (personal opinion, rant over!)

For this reason, I think all `Tweak`s would be better as `Dev` - i.e. we did it to make the code base better to work on, and/or for developers working with the plugin. 

Also I think it's useful to distinguish between fixes and new features – can we add `New` or `Feature`?

Reply:
> For this reason, I think all Tweaks would be better as Dev - i.e. we did it to make the code base better to work on, and/or for developers working with the plugin.

Thank you for explaining your thoughts; what you said makes sense to me. I have removed `Tweak` and leave `Dev`.

> Also I think it's useful to distinguish between fixes and new features – can we add New or Feature?

Word. added as `New` as can cover not only new (new) features